R/config.R

```diff
@@ -120,34 +120,26 @@ validate_config_targets <- function(webevals_config, task_groups, task_id_names)
     }
 
     # check that metrics are valid for the available output types
-    output_types_for_target <- purrr::map(
```

Comment:
moved computation of `output_types_for_target` and `target_is_ordinal` into their own functions in `utils-hub_tasks_config.R`.  These functions are now called from within `get_metric_name_to_output_type`.

Slightly awkwardly, to support issuing informative errors, we also compute them below if we need to issue the error.